elife-1234567890-v2/elife-1234567890-v2.xml

```diff
@@ -192,37 +192,46 @@
             <pub-date publication-format="electronic" date-type="publication">
                 <day>22</day>
                 <month>10</month>
-                <year>2020</year>
+                <year>2023</year>
             </pub-date>
             <pub-date pub-type="collection">
-                <year>2020</year>
+                <year>2023</year>
             </pub-date>
-            <volume>9</volume>
+            <volume>12</volume>
             <elocation-id>e00666</elocation-id>
             <history>
-                <date date-type="received" iso-8601-date="2020-03-20">
+                <date date-type="received" iso-8601-date="2023-03-20">
                     <day>20</day>
                     <month>03</month>
-                    <year>2020</year>
+                    <year>2023</year>
                 </date>
-                <date date-type="accepted" iso-8601-date="2020-09-18">
-                    <day>18</day>
-                    <month>09</month>
-                    <year>2020</year>
+                <date date-type="reviewed" iso-8601-date="2023-03-20">
+                    <day>10</day>
+                    <month>04</month>
+                    <year>2023</year>
                 </date>
             </history>
             <pub-history>
                 <event>
                     <event-desc>This manuscript was published as a preprint at bioRxiv.</event-desc>
-                    <date date-type="preprint" iso-8601-date="2019-02-15">
+                    <date date-type="preprint" iso-8601-date="2023-02-15">
                         <day>15</day>
                         <month>02</month>
-                        <year>2019</year>
+                        <year>2023</year>
                     </date>    
                     <!-- Should this be content-type="preprint" or content-type="pre-print"? Examples and text in JATS4R conflict
                             https://jats4r.org/article-publication-and-history-dates/ -->
                     <self-uri content-type="preprint" xlink:href="https://doi.org/10.1101/2021.11.09.467796"/>   
                 </event>
+                <event>
+                    <event-desc>This manuscript was published as a reviewed preprint at ????.</event-desc>
+                    <date date-type="preprint" iso-8601-date="2023-04-10">
+                        <day>10</day>
+                        <month>04</month>
+                        <year>2023</year>
+                    </date>
+                    <self-uri content-type="preprint" xlink:href="https://doi.org/?????????"/>   
```

Comment:
Would need to be a URL rather than a DOI (since the VOR would overwrite the reviewed preprint DOI). The URL is as yet unknown, so we can just add some dummy URL for now.